android/anc/src/main/java/org/smartregister/fhircore/anc/AncApplication.kt

```diff
@@ -45,6 +48,10 @@ open class AncApplication : Application(), ConfigurableApplication {
   override val secureSharedPreference: SecureSharedPreference
     get() = SecureSharedPreference(applicationContext)
 
+  override val authenticatedUserInfo: UserInfo?
+    get() =
+      SharedPreferencesHelper.read(USER_INFO_SHARED_PREFERENCE_KEY, null)?.decodeJson<UserInfo>()
```

Comment:
Do we need to override this from the base class?

Reply:
Its interface so we need to provide implementation of how app identifies the authenticated user. This would be used across app to pick logged-in user info when needed